test/DynamoCoreWpfTests/PreviewBubbleTests.cs

```diff
@@ -188,9 +190,9 @@ public void PreviewBubble_CodeBlockIsNotInFocus()
             // Click on DragCanvas.
             View.ChildOfType<DragCanvas>().RaiseEvent(
                 new MouseButtonEventArgs(Mouse.PrimaryDevice, 0, MouseButton.Left)
-            {
-                RoutedEvent = Mouse.MouseDownEvent
-            });
+                {
```

Comment:
Minor: Some alignment issues, with extra spacing and lines.

Reply:
Done.